libs/top_level_metrics.py

```diff
@@ -100,14 +102,35 @@ def calculate_metrics_for_timeseries(
 
     metric_summary = None
     if not metrics.empty:
-        metric_summary = calculate_latest_metrics(metrics, icu_metric_details)
+        metric_summary = calculate_latest_metrics(
+            metrics, icu_metric_details, test_positivity_details
+        )
 
     return metrics, metric_summary
 
 
-def calculate_or_copy_test_positivity(data):
+def _lookup_test_positivity_method(
+    positive_tests_provenance: Optional[str], negative_tests_provenance: Optional[str], log
+) -> TestPositivityRatioMethod:
+    method = None
+    if positive_tests_provenance and positive_tests_provenance == negative_tests_provenance:
+        method = TestPositivityRatioMethod.get(positive_tests_provenance)
+    if method is None:
+        log.info(
+            "Unable to find TestPositivityRatioMethod",
```

Comment:
After looking at the logs, looks like this is pretty noisy.  We should probably remove this log or maybe find a way to log total number of "other" results

Reply:
Cleaning this up will be much easier when test positivity calculation is extracted from top_level_metrics. Changed to debug for now.